src/components/CurrencyLogo/index.tsx

```diff
@@ -55,6 +56,8 @@ export default function CurrencyLogo({
     return <StyledEthereumLogo src={AvaxLogo} size={`${size}px`} style={style} />
   } else if (chainId && currency === CAVAX[ChainId.WAGMI]) {
     return <StyledEthereumLogo src={WgmLogo} size={`${size}px`} style={style} />
+  } else if (chainId && currency === CAVAX[ChainId.COSTON]) {
+    return <StyledEthereumLogo src={CostonLogo} size={`${size}px`} style={style} />
```

Comment:
please update same logo in components PR also. https://github.com/pangolindex/components/pull/18

Reply:
@0xAtyo will update his PR according

Reply:
Updated ✅